Package.swift

```diff
@@ -60,7 +60,14 @@ let package = Package(
         .testTarget(name: "TrackingTests",
                     dependencies: ["CioTracking", "SharedTests"],
                     path: "Tests/Tracking"),
-
+        // Migration
+        // this module handles Journeys tasks migration to Datapipeline.
+        .target(name: "CioMigration",
```

Comment:
If we are creating a new target, [we will want to add the target name to the product list above in the file](https://github.com/customerio/customerio-ios/pull/530/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR31) to make sure that code test coverage is captured.

Reply:
`CioMigration` sound too generic, maybe `CioTrackingMigration` ?

Reply:
I did not want to include the word `Tracking` because we are deprecating the tracking module and adding the word tracking would sound like the migration still supports tracking ! It is a personal preference of the module name, we can change it if the majority of the team feels alike !

Reply:
Two votes for `CioTrackingMigration`. Name has been updated ! 🥳